besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1882,6 +1882,16 @@ private PrivacyParameters privacyParameters() {
         throw new ParameterException(
             commandLine, "Please specify Enclave public key file path to enable privacy");
       }
+
+      if (Wei.ZERO.compareTo(minTransactionGasPrice) < 0) {
+        // if gas is required, cannot use random keys to sign private tx
+        // ie --privacy-marker-transaction-signing-key-file must be set
+        if (privacyMarkerTransactionSigningKeyPath == null) {
+          throw new ParameterException(
+              commandLine,
+              "Not a free gas network. --privacy-marker-transaction-signing-key-file must be specified and must be a funded account. Private transactions cannot be signed by random (non-funded) accounts in paid gas networks");
```

Comment:
"Private transactions cannot be signed by random (non-funded) accounts in paid gas networks"

Would you expect a user to connect that not specifying a key file will mean a random account is used? I'm not sure whether we can expect the user to have enough context to understand the last sentence.

Reply:
I got that from reading the docs. I think if you're using privacy you should read the docs. Also I asked @MadelineMurray to preapprove this error message